### PR TITLE
tests: Consistently use `store::`

### PR DESCRIPTION
**Pull request by cgwalters**
Seeing the full `ostree_ext::container::store::` a lot is too
verbose.  Consistently shorten this to just `store::`.

### CODE REVIEW

File: lib/tests/it/main.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ use cap_std::fs::{Dir, DirBuilder};
 use once_cell::sync::Lazy;
 use ostree::cap_std;
 use ostree_ext::chunking::ObjectMetaSized;
-use ostree_ext::container::store::PrepareResult;
+use ostree_ext::container::store;
```
Comment (lucab) [inline lib/tests/it/main.rs:7]: It looks like you maybe wanted `ostree_ext::container::store::{self, ImageImporter, PrepareResult}`? But this way is ok too.

**Review by lucab — APPROVED** [review]

File: lib/tests/it/main.rs (hunk 1 above)
Comment (cgwalters) [reply lib/tests/it/main.rs:7]: Hmm...yeah, may consider that in a followup.
